src/navigation/types.ts

```diff
@@ -0,0 +1,70 @@
+import { createStackNavigator } from '@react-navigation/stack';
+
+import Routes from '@/navigation/routesNames';
+
+import { PortalSheetProps } from '@/screens/Portal';
+import { REGISTRATION_MODES } from '@/helpers/ens';
+
+export type PartialNavigatorConfigOptions = Pick<
+  Partial<Parameters<ReturnType<typeof createStackNavigator>['Screen']>[0]>,
+  'options'
+>;
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace ReactNavigation {
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface RootParamList extends RootStackParamList {}
+  }
+}
```

Comment:
This does some light typing for us when using the exported `useNavigation` hooks and such